src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -145,9 +145,9 @@ module sonic-port{
 
 				leaf role {
 					type string {
-						pattern "Ext|Int|Inb|Rec";
+						pattern "Ext|Int|Inb|Rec|Dpc";
```

Comment:
@vivekrnv any specific reason to not use `Dpu` ? If not can we change to `Dpu`?

Reply:
'Dpc' role applies for the ports on the DPU side as well. So, think "DPU Connect port" is a better name